### ISSUE TITLE
[Bug]: Description now being hidden

### ISSUE THREAD

**Issue by GGhi2** [sapondanaisriwan/AdashimaaTube#175]
### Describe the bug

The description has fully disappeared. 

### Screenshots

![image](https://github.com/sapondanaisriwan/AdashimaaTube/assets/148836730/334530d1-a0cc-4393-a81a-ea6d53419dc0)

### What browser are you using?

Google chromeos Version 116.0.5845.210 (Official Build) (64-bit)

### What version of AdashimaaTube are you using?

v2.4.4

**Comment by MrGoatsy** [comment]
Same here.

**Comment by sapondanaisriwan** [comment]
https://github.com/sapondanaisriwan/AdashimaaTube#-installation
![image](https://github.com/sapondanaisriwan/AdashimaaTube/assets/64634605/6bbf65cb-d6d0-4d66-8f40-39351b83753a)

**Comment by Mirrorman95** [comment]
I'm also getting the Descriptions hidden.

**Comment by MrGoatsy** [comment]
> I'm also getting the Descriptions hidden.
> 
> EDIT: I came up with a quick fix:
> 
>     1. Open Stylus settings
> 
>     2. Installed Styles
> 
>     3. Press the gear icon just to the right of AdashimaaTube
> 
>     4. Under Old Layout, set the Old Watchpage slider from "Enabled" to "Disabled".

That works, but now the description sticks out due to the light grey on black background.

**Comment by Usv3re** [comment]
Same problem. Thnx **Mirrorman95** good for now

Time to update Adashima though, it seems

**Comment by Mirrorman95** [comment]
> Same problem. Thnx **Mirrorman95** good for now
> 
> Time to update Adashima though, it seems
You're welcome.

**Comment by re11ding** [comment]
Yep, here to confirm for the same issue. Thanks for the temp fix though.

**Comment by Denver5280** [comment]
Description hdden in Brave and Vivaldi
![Op28No20 1](https://github.com/sapondanaisriwan/AdashimaaTube/assets/131574099/18f3a83d-2121-4829-959e-4d856f81d656)
AdashimaaTube 2.4.4

**Comment by sliceanddice9** [comment]
> https://github.com/sapondanaisriwan/AdashimaaTube#-installation ![image](https://user-images.githubusercontent.com/64634605/276508767-6bbf65cb-d6d0-4d66-8f40-39351b83753a.png)

This is working a treat with bringing back the old description box for the old UI. Thank You.

**Comment by sliceanddice9** [comment]
> https://github.com/sapondanaisriwan/AdashimaaTube#-installation ![image](https://user-images.githubusercontent.com/64634605/276508767-6bbf65cb-d6d0-4d66-8f40-39351b83753a.png)

This works a treat with bringing back the description box for the old UI, but after doing this I am noticing that the 'Save' button is no longer present on-screen, the 'Save' button is instead being redirected to the overflow menu. Is there something I can do to make the 'Save' button back to being on-screen at all times for all videos?

**Comment by rakitanc** [comment]
Same here

**Comment by idkagithubusername** [comment]
> https://github.com/sapondanaisriwan/AdashimaaTube#-installation ![image](https://user-images.githubusercontent.com/64634605/276508767-6bbf65cb-d6d0-4d66-8f40-39351b83753a.png)

this method does not work for firefox version 115.4.0esr (64-bit)

**Comment by sapondanaisriwan** [comment]
I will continue on this project after I finish the exam. I may take 2 months.
